TLM/TLM/UI/Helpers/SerializableUIOptionBase.cs

```diff
@@ -9,34 +9,76 @@ namespace TrafficManager.UI.Helpers {
 
     public abstract class SerializableUIOptionBase<TVal, TUI> : ILegacySerializableOption
         where TUI : UIComponent {
-        //Data:
-        public SerializableUIOptionBase(string fieldName) {
+
+        /* Data: */
+        public SerializableUIOptionBase(string fieldName, Options.PersistTo scope) {
+
             ValueField = typeof(Options).GetField(fieldName, BindingFlags.Static | BindingFlags.Public);
             if (ValueField == null) {
-                throw new Exception($"{typeof(Options)}.{fieldName} does not exists");
+                throw new Exception($"{typeof(Options)}.{fieldName} does not exist");
             }
+
+            Scope = scope;
         }
+
+        public TranslatorDelegate Translator;
```

Comment:
Maybe turn this into a property.
```
        private TranslatorDelegate _translator;
        public TranslatorDelegate Translator {
            get => _translator ?? Translation.Options.Get;
            set => _translator = value;
        }
        protected string T(string key) => Translator(key); // we can do away with T actually.
```

Reply:
Good idea! Will probably keep `T()` as shortcut.